test/factories/factories.rb

```diff
@@ -29,7 +33,7 @@
     end
   end
 
-  sequence :name do |n|
+  sequence :name, ['a', 'b', 'c'].cycle do |n|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%w` or `%W` for array of words.